setup.php

```diff
@@ -374,7 +374,7 @@ function plugin_formcreator_hook(): void {
    $PLUGIN_HOOKS[Hooks::REDEFINE_MENUS]['formcreator'] = [PluginFormcreatorCommon::class, 'hookRedefineMenu'];
 
    // Config page
-   if (Session::haveRight('entity', UPDATE)) {
+   if (Session::haveRight(PluginFormcreatorForm::$rightname, UPDATE)) {
```

Comment:
With your feature I think that we should check if the user has at least one right among CREATE, READ, UPDATE, DELETE, PURGE and maybe the import right if we implement it (see comments above)